solutions/instances/variables.tf

```diff
@@ -123,6 +135,22 @@ variable "cloud_monitoring_service_endpoints" {
   }
 }
 
+########################################################################################################################
+# COS targets and AT routes
+########################################################################################################################
+
+variable "provision_cos_targets" {
+  description = "Set it to true if cos targets is to be provisioned"
```

Comment:
`description = "Set it to true if AT targets and routes needs to be configured."`